strapi/strapi
# feat(cli): use project name from package json by default

### What does it do?

When prompting the user for the name of the project during the deployment process, the setDefaultName function sets the default project name to the name of the existing Strapi project (retrieved from package.json), instead of using a generic default name like "my-strapi-project".

- Also added unit testing config to the cli/cloud package.
To run: `yarn test:unit` from the cloud folder.

### Why is it needed?

This change is needed to improve the user experience during the deployment process

### How to test it?

- you need to have Strapi Cloud services running locally (main server + cli-api)
- run `yarn build` in the strapi/strapi folder containing this code's repository
- create a new strapi CMS project
- from the strapi CMS project: call the built `strapi.js` command to run the `login` command
- Then, use the `deploy` command:
When asked for the name of the project, check that the default value is the name of your existing project (as specified in package.json), not "my-strapi-project".

Result:

<img width="681" alt="Screenshot 2024-06-14 at 11 28 35" src="https://github.com/strapi/strapi/assets/55534657/6878037c-d820-4229-b2cf-5fc1e297e9a1">


### Related issue(s)/PR(s)

Let us know if this is related to any issue/pull request
